addons/account/controllers/portal.py

```diff
@@ -49,7 +49,7 @@ def portal_my_invoices(self, page=1, date_begin=None, date_end=None, sortby=None
         order = searchbar_sortings[sortby]['order']
 
         searchbar_filters = {
-            'all': {'label': _('All'), 'domain': [('move_type', 'in', ['in_invoice', 'out_invoice'])]},
+            'all': {'label': _('All'), 'domain': [('move_type', '!=', 'entry')]},
             'invoices': {'label': _('Invoices'), 'domain': [('move_type', '=', 'out_invoice')]},
             'bills': {'label': _('Bills'), 'domain': [('move_type', '=', 'in_invoice')]},
```

Comment:
```suggestion
            'all': {'label': _('All'), 'domain': []},
            'invoices': {'label': _('Invoices'), 'domain': [('move_type', 'in', ('out_invoice', 'out_refund')]},
            'bills': {'label': _('Bills'), 'domain': [('move_type', 'in', ('in_invoice', 'in_refund'))]},
```

Sorry for the back and forth

Reply:
Done :)